### DISCUSSION TITLE
How to unpublish a `Codemagic Static Pages` website ?

### DISCUSSION THREAD

**Discussion by divyanshub024** [codemagic-ci-cd/codemagic-docs#2221] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Publish a flutter web app using Codemagic Static Pages 
https://docs.codemagic.io/flutter-publishing/publishing-to-codemagic-static-pages/

### Expected results

An ability to delete/unpublish the webapp from the Codemagic Static Page. 

### Actual results

No such way to do that

**Comment by dtrdic** [comment, marked as answer]
Hello @divyanshub024 unfortunately, there's currently no way for you to delete it yourself. I will take it down from our end. Please share the URL of your static page so that I can remove it

**Reply by divyanshub024** [reply]
Thanks @dtrdic. Here is the URL https://mweb.codemagic.app/

**Reply by dtrdic** [reply]
@divyanshub024 done

**Reply by bhaumik1111** [reply]
hi same problem here user is getting use of that testing page so i want disable that page and then they can refer new site.?
how can i do it?

**Reply by dtrdic** [reply]
@bhaumik1111 please share your web page subdomain name so i can take it down

**Reply by bhaumik1111** [reply]
Hello, @dtrdic 
What if i want to enable this domain after few months of week.
Its easy to enble it right ?
Or i have to contact to support team to make domain enable?

**Reply by kunalmnnit** [reply]
Could you please do this for me as well
https://kunalmnnit.codemagic.app/#/

**Reply by dtrdic** [reply]
@kunalmnnit done.

**Reply by bhaumik1111** [reply]
please reply me on this!!! @dtrdic
Hello, @dtrdic
What if i want to enable this after few months of week.
Its easy to enble it right ?
Or i have to contact to support team to make domain enable?

**Reply by dtrdic** [reply]
hey @bhaumik1111 yes it's easy to create and submit the app to a web subdomain, but once the static page is unpublish all data is wiped out from there.

**Reply by divyanshub024** [reply]
@dtrdic It would be great to add the unpublish feature in Codemagic.

**Reply by bhaumik1111** [reply]
@dtrdic 
hi here is my codemagic url which i want to disable !
https://supremetoday.codemagic.app/

**Reply by dtrdic** [reply]
@bhaumik1111 done

**Reply by NeyDavid** [reply]
@dtrdic 
hi, can you do the same with this website please.

https://backoffice.codemagic.app/

**Reply by dtrdic** [reply]
@NeyDavid done

**Reply by dkinvoker1** [reply]
Hi,
Is it still the case?
If yes, can you disable both
https://sum_bw.codemagic.app/
https://sum_bw_admin.codemagic.app/#/session-aware-route

**Reply by dtrdic** [reply]
@dkinvoker1 done
